test/rpc-integration/test-diamond/response-mutual-constraints-stuck.booster-dev

```diff
@@ -97,15 +97,218 @@
             }
         },
         "logs": [
+            {
+                "tag": "rewrite",
+                "origin": "booster",
+                "result": {
+                    "tag": "success",
```

Comment:
When using an SMT-enabled server, the difference is actually only in the rewrite log - maybe we switch off the log and delete the extra files?